applications/openshift/kubelet/kubelet_eviction_thresholds_set_hard_imagefs_inodesfree/rule.yml

```diff
@@ -55,9 +54,12 @@ identifiers:
   cce@ocp4: CCE-84147-8
 
 references:
+  cis@ocp4: 1.3.1
```

Comment:
`1.3.1` was according to CIS v1.3.0.
In CIS v1.4.0, it should be `4.2.1`, but we didn't select it.
Was it intentional?

Reply:
this was intentional, we are only check if those default kubeletconfig garbage collection values are present:
{
"imagefs.available": "15%",
"memory.available": "100Mi",
"nodefs.available": "10%",
"nodefs.inodesFree": "5%"
}

Reply:
https://github.com/Vincent056/cac-content-fork/commit/37800db7c83640042e6c7e08862ebb7d491814b7#diff-26f0beba8278470e3ef6ffbfd532e66dbf860df91e1e2d66f7801a57e41aa5faR73

Reply:
@Vincent056 So I think this rule should have its `cis` reference removed.